src/app.css

```diff
@@ -132,7 +133,7 @@ body {
 
 /* Variables */
 body {
-  font-family: "Dosis, sans-serif";
+  font-family: Inter, sans-serif;
```

Comment:
Why are we loading Dosis then?

Reply:
In Figma every text I find is Inter. Therefore I thought it would be good to add this as the fallback for all cases where we forget to use <Typography>. We should probably not load Dosis then. @kommander ?

Reply:
Then we should add Inter font instead of Dosis.